.env.example

```diff
@@ -0,0 +1,5 @@
+# -------------- Integration stuff --------------
+
+### LOCAL (optimism-integration repo)
+L1_WEB3_URL=http://0.0.0.0:9545
+L2_WEB3_URL=http://0.0.0.0:8545
```

Comment:
Is `WEB3_URL` the right name for this endpoint? Maybe `ETHEREUM_JSON_RPC_ENDPOINT`?